projects/plugins/boost/changelog/update-boost-add-tracks-events

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: added
+
+General: add Tracks events to clickable events on settings page
```

Comment:
Did you mean `clickable elements` instead of `clickable events`?

Reply:
Yes, thanks!